pypif/obj/common/property.py

```diff
@@ -45,8 +47,10 @@ def __init__(self, name=None, scalars=None, vectors=None, matrices=None, files=N
         self.licenses = licenses
         self._conditions = None
         self.conditions = conditions
-        self._method = None
-        self.method = method
+        self._methods = None
+        self.methods = methods
+        if 'method' in kwargs:
+            self._methods = method
```

Comment:
This should be `self.methods = method` so that it calls the setter method, not `self._methods = method`